scripts/validate/govcms-validate-active-modules

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+#
+# GovCMS mandatory module validation.
+#
+# Ensure requirde platform modules are enabled.
+#
+
+GOVCMS_PREPARE_XML_SCRIPT=${GOVCMS_PREPARE_XML_SCRIPT:-govcms-prepare-xml}
+GOVCMS_OUTFILE=${GOVCMS_OUTFILE:-govcms-validate-active-permissions}
+GOVCMS_REMEDIATE=${GOVCMS_REMEDIATE:-""}
+
+GOVCMS_REQUIRED_MODULES=(
+  tfa
+  govcms_security
+)
+
+GOVCMS_DISALLOWED_MODULES=(
+  update
+  dblog
+)
+
+FAILURES=""
+TOTAL=0
+
+echo "GovCMS Validate :: Active modules validation"
+
+MODULE_LIST=$(drush pm:info --status=enabled --format=json)
+
+# Check if required modules are enabled
+for module in "${GOVCMS_REQUIRED_MODULES[@]}"; do
+  (( TOTAL=TOTAL+1 ))
+  if [[ 'null' == $(jq -r ".$module" <<< "${MODULE_LIST}") ]]; then
+    if [ -n "${GOVCMS_REMEDIATE}" ]; then
+      drush pm:enable "$module" -y
+    else
+      echo "[fail]: '$module' is not enabled";
+      FAILURES="$FAILURES,disabled:$module"
+    fi
+  fi
+done
+
+# Check if disallowed modules are disabled
+for module in "${GOVCMS_DISALLOWED_MODULES[@]}"; do
+  (( TOTAL=TOTAL+1 ))
+  if [[ 'null' != $(jq -r ".$module" <<< "${MODULE_LIST}") ]]; then
+    if [ -n "${GOVCMS_REMEDIATE}" ]; then
+      drush pm:disable "$module" -y
```

Comment:
Prepares support for the scheduled run to enable/disable these modules based on an env var.